gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Skin.java

```diff
@@ -525,9 +527,24 @@ public Object read (Json json, JsonValue jsonData, Class type) {
 				return drawable;
 			}
 		});
+		
+		for (Class cls : AUTO_TAGGED_STYLES){
+			json.addClassTag(cls.getSimpleName(), cls);
+		}
 
 		return json;
 	}
+	
+	private static final Class[] AUTO_TAGGED_STYLES = {

```

Comment:
This may be better as an ObjectMap<String, Class> field on Skin, which would allow users to customize the tags before loading Skin JSON. It could be populated with reasonable defaults.